src/glmfit.jl

```diff
@@ -296,6 +296,19 @@ function StatsBase.fit!(m::AbstractGLM, y; wts=nothing, offset=nothing, dofit::B
     end
 end
 
+"""
+    fit(AbstractGLM, X, y, d, [l = canonicallink(d)]; <keyword arguments>)
+
+Fit a generalized linear model to data.
+
+# Keyword Arguments
+- `verbose::Bool=false`: Display convergence information for each iteration
+- `maxIter::Integer=30`: Maximum number of iterations allowed to achieve convergence
+- `convTol::Real=1.e-6`: Convergence is achieved when the relative change in
+deviance is less than this
+- `minStepFac::Real=0.001`: Minimum line step fraction. Must be between 0 and 1.
+- `start=nothing`: Starting values for beta
```

Comment:
Can you give the type of `start` and its required dimension?

Reply:
Done